_posts/2020-08-04-newpipe-turning-5.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Happy five Birthday NewPipe!!!
+short: Happy Birthday NewPipe!!!
+date: 2020-08-05 05:00:00 +0100
+author: Schabi
+image: birthday-cake
+---
+
+Dear NewPipers, NewPipe Users, Hello World!
+
+It's a weird time right now. Many things have changed and don't work like they used to before. However, NewPipe and the NewPipe project just continue like nothing happens. NewPipe constantly gets bigger and better. People keep working on it and organize the project like COVID-19 doesn't exist. For example, just a week ago we had the very first developer video call and, just one week before that, NewPipe hit number one trending on Hackernews. Next to all of these things going on one might be tempting to forget: NewPipe is turning five years old today!
+
+Yes, it's true! Our little project that started as a simple tool to solve a simple problem is already this old!
+On September 4th, 2015 I uploaded v0.3 as a tag on GitHub. By this date NewPipe was really simple and small and could not do much, but already existed for about a month until I finally decided to push the release. Little did I know this would turn out to be such an amazing project with many amazing people who create such an awesome tool.
```

Comment:
Nope, you did on [September 15th](https://github.com/TeamNewPipe/NewPipe/releases/tag/v0.3)

But the initial commit was on [September 4th](https://github.com/TeamNewPipe/NewPipe/commit/73d61f17b55f23951dc60dc9fbd469c99fb0adaa)

Reply:
Are you sure?

![](https://i.ibb.co/r028QXN/Screenshot-20200904-181706-Firefox.png)

Reply:
The tag was likely created before the release (i.e., binaries were published).

Reply:
really weird. But yeah, it's very unlikely that 0.3.5 was released one day before 0.3

Reply:
It's probably because the tag kept existing on Git, but the release was (perhaps accidentally) removed and then recreated on 15 September. Back then no binaries where uploaded to GitHub and iirc v0.3.5 is the first release that was published on F-Droid.